ephios/urls.py

```diff
@@ -28,7 +27,6 @@
     path("", include("ephios.core.urls")),
     path("api/oauth/", include("ephios.api.access.oauth2_urls", namespace="oauth2_provider")),
     path("api/", include("ephios.api.urls")),
-    path("jsi18n.js", JavaScriptCatalog.as_view(packages=["recurrence"]), name="jsi18n"),
```

Comment:
where does this come from now instead?

Reply:
Actually, it was never used! There are two options to deliver the i18n js:

1. Djangos built-in solution is to use the `JavaScriptCatalog`, include it in the urls as above **and** load it in the template (which we never did, because we use option 2).
2. `django-statici18n`, which collects the catalog for every language into a folder so it does not have to be generated on the fly for every (non-cached) request.

With option 2, we can include the catalog in compression.